crates/figma_import/src/transform_flexbox.rs

```diff
@@ -214,6 +214,21 @@ fn compute_layout(node: &Node, parent: Option<&Node>) -> ViewStyle {
                     }
                 }
             }
+
+            if frame.layout_mode != LayoutMode::None {
+                let width_points = bounds.width().ceil();
+                let height_points = bounds.height().ceil();
+                style.width = match frame.layout_sizing_horizontal {
+                    LayoutSizing::Fixed => Dimension::Points(width_points),
+                    LayoutSizing::Fill => Dimension::Points(width_points),
```

Comment:
Does it work if you just set min_width / min_height? (You probably tried that...).

Reply:
Nope that doesn't work, that makes the size grow too large also. I'm thinking I may have to make autolayout frames with children that have a FILL sizing mode to somehow ignore the children when calculating layout size.

Reply:
Hm, ok, this is another dumb idea, but what are the default settings for flex and align-items? I'm wondering if there's some default style value that's set inappropriately.

Reply:
flex grow/shrink both default to 0, and align-items defaults to Stretch. However align-items should get set to something else if it is a frame, based on the value of counter_axis_align_items. I can dig it into it more and try and reproduce with a flexbox example somewhere else.